azure/packages/azure-service-utils/src/generateToken.ts

```diff
@@ -3,7 +3,8 @@
  * Licensed under the MIT License.
  */
 
-import type { ITokenClaims, IUser, ScopeType } from "@fluidframework/protocol-definitions";
+import type { IUser } from "@fluidframework/driver-definitions";
+import type { ITokenClaims, ScopeType } from "@fluidframework/driver-definitions/internal";
```

Comment:
Moving the types to 'driver-definitions' forces us to update imports to respect tagging.

This happened because 'protocol-definitions' has been tagged and trimmed, but we never published/consumed the latest version on the client, so the imports hadn't been updated to point to '/' vs. '/internal'.

Imports were updated using `flub modify fluid-imports --onlyInternal --packageRegex=driver-definitions`